autoload/jetpack.vim

```diff
@@ -506,23 +509,15 @@ function! jetpack#end() abort
 endfunction
 
 function! jetpack#tap(name) abort
-  for pkg in s:packages
-    if pkg.name ==# a:name
-      return isdirectory(pkg.path)
-    endif
-  endfor
-  return 0
+  return has_key(s:packages, a:name) ? isdirectory(jetpack#get(a:name).path) : 0
```

Comment:
`... ? ... : ...` is redundant.  Consider to use `&&` instead of that.

Reply:
I think it is readable, but if you think it is redundant, I'll fix it.